RecyclerBot/Assets/Scripts/ThrowBall.cs

```diff
@@ -14,6 +14,8 @@ public class ThrowBall : MonoBehaviour
     private Camera mainCamera;
     private Vector2 force;
     private float torque = 10f;
+    private float throwingAreaBorderRight = -6.0f;
```

Comment:
Instead of hard-coding the values here, could we compute them as an offset from the `startPosition` in the `Start()` method?